sos/report/plugins/ipmitool.py

```diff
@@ -26,12 +26,14 @@ def setup(self):
         if result['status'] == 0:
             cmd += " -I usb"
 
+        for subcmd in ['channel info', 'channel getaccess', 'lan print']:
+            for channel in [1, 3]:
```

Comment:
Is the use of channel 1 Dell specific? Or can we expect other vendor's hardware to respond to these commands as well?

Reply:
It's not Dell specific. Every vendor can choose. Others can choose same number.

Reply:
@TurboTurtle , any further thoughts?

Reply:
Please add a bit more descriptive text to the commit message. Dell being singled out in the title is what made me ask if channel 1 use was Dell-specific, and some context for when someone looks back at the git history would be helpful.

Reply:
Sure, I will update the commit message. Remove reference of Dell from the first line and add some details in the description. Let me know if you are looking for any specific text that you want me to add.

Reply:
Updated the commit message.